pkg/kv/kvserver/batcheval/cmd_add_sstable_test.go

```diff
@@ -928,12 +945,34 @@ func TestEvalAddSSTable(t *testing.T) {
 						}
 						require.NoError(t, b.Commit(false))
 						stats := storageutils.EngineStats(t, engine, 0)
+						// All timestamps are experienced in increments of 1e9. This is
+						// to accurately test for GCBytesAge in stats, which is only
+						// calculated in increments of 1e9.
+						tc.toReqTS *= 1e9
+						tc.reqTS *= 1e9
+						tc.noShadowBelow *= 1e9
 
 						// Build and add SST.
 						if tc.toReqTS != 0 && tc.reqTS == 0 && tc.expectErr == nil {
 							t.Fatal("can't set toReqTS without reqTS")
 						}
-						sst, start, end := storageutils.MakeSST(t, st, tc.sst)
+						var sstKvs []interface{}
+						for i := range tc.sst {
```

Comment:
nit: mild preference for just using `e9` in the original timestamps -- it's obvious, and doesn't need any additional code.